src/components/CookieApprovalDialog.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import Cookies from 'js-cookie';
+import Button from './Button';
+import styles from './CookieApprovalDialog.module.scss';
```

Comment:
We _could_ move to emotion for the styles. We could also totally wait.

Reply:
I want to wait on that for this PR, but soon!